test/selecting-items.html

```diff
@@ -25,36 +25,52 @@
     var combobox;
     var input;
     var selector;
+    var valueChangedSpy;
 
     beforeEach(function() {
       combobox = fixture('combobox');
-      input = combobox.$$("input");
+      combobox.items = ['foo', 'bar'];
```

Comment:
Do we have some coding conventions about these, btw? Would be nice if everyone used the same (and if there was a linter that warned about using the incorrect ones).